server/auth/openai_oauth_browser_test.go

```
@@ -87,6 +88,44 @@ func TestStartOAuthCallbackListenerUsesLocalhostAuthCallback(t *testing.T) {
 
+func TestOAuthCallbackSuccessResponseServesAuthCompleteHTML(t *testing.T) {
+	listener, err := StartOAuthCallbackListener()
+	if err != nil {
+		if strings.Contains(strings.ToLower(err.Error()), "address already in use") {
+			t.Skipf("oauth callback port in use: %v", err)
+		}
+		t.Fatalf("start listener: %v", err)
+	}
+	defer listener.Close()
+
+	resp, err := http.Get(listener.RedirectURI() + "?code=auth-code&state=state-1")
```

golangci-lint (2.12.2)

[error] net/http.Get must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)